packages/vitest/src/constants.ts

```diff
@@ -1,11 +1,37 @@
 import { fileURLToPath } from 'url'
 import { resolve } from 'pathe'
+import type { UserConfig } from './types'
+import { defaults as coverageDefaults } from './integrations/coverage'
 
 export const distDir = resolve(fileURLToPath(import.meta.url), '../../dist')
 
 export const defaultInclude = ['**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}']
 export const defaultExclude = ['**/node_modules/**', '**/dist/**', '**/cypress/**', '**/.{idea,git,cache,output,temp}/**']
 
+export const defaults: UserConfig = {
+  globals: false,
+  environment: 'node',
+  threads: true,
+  clearMocks: false,
+  restoreMocks: false,
+  mockReset: false,
+  include: defaultInclude,
+  exclude: defaultExclude,
+  testTimeout: 5_000,
+  hookTimeout: 10_000,
+  isolate: true,
+  watchIgnore: [/\/node_modules\//, /\/dist\//],
+  update: false,
+  watch: true,
+  reporters: 'default',
+  silent: false,
+  api: false,
+  ui: false,
+  uiBase: '/__vitest__/',
+  open: true,
+  coverage: coverageDefaults,
+}
```

Comment:
We might also want to froze this object to avoid mutation on user land

Reply:
Froze both objects